src/metal.js

```diff
@@ -1,14 +1,49 @@
 // Missing Metal Items
-// 1. Marionette.Class parity
+// 1. Marionette.Error
 // 2. Marionette.Deprecate
 
-Marionette.Class = Metal.Class.extend({});
+Marionette.Class = Metal.Class.extend({
+  constructor: function(options) {
+    this.options = _.extend({}, _.result(this, 'options'), options);
+    Marionette.proxyRadioHandlers.apply(this);
+    this.cid = _.uniqueId(this.cidPrefix);
+    this.initialize.apply(this, arguments);
+  },
+
+  cidPrefix: 'mnc',
+
+  destroy: function() {
+    this.triggerMethod('before:destroy');
+    this.triggerMethod('destroy');
+    Marionette.unproxyRadioHandlers.apply(this);
```

Comment:
With `Metal` it'd be cool if the radio stuff was just a mixin.  I dunno that it needs to be available on every single Mn and Bb class there is. But that would require https://github.com/marionettejs/backbone-metal/issues/20  or you'd have to init the radio handler mixin yourself.